minard/cssProc.py

```diff
@@ -0,0 +1,117 @@
+import contextlib
+import shutil
+import tempfile
+import argparse
+import sys
+import os
+import os.path
+import time
+import re
+import string
+import subprocess
+import datetime
+import json
+
+import psycopg2
+import psycopg2.extras
+import psycopg2.extensions
+
+from .db import engine_nl
+from .views import app
+
+
+RUN_TYPES = {
+    0:"Maintenance",
+    1:"Transition",
+    2:"Physics",
+    3:"Deployed Source",
+    4:"External Source",
+    5:"ECA",
+    6:"Diagnostic",
+    7:"Experimental",
+    8:"Supernova",
+    }
+
+def getStandardRunList():
+    try:
+        os.environ['PGOPTIONS'] = '-c statement_timeout=10000' #in ms
+        conn = psycopg2.connect('dbname=%s user=%s host=%s connect_timeout=5 password=%s '
```

Comment:
I think it would be best to use the connection to the nearline database which is already set up for minard. To see an example check out https://github.com/snoplus/minard/blob/master/minard/muonsdb.py.

Reply:
I think you are right i should use the connection built and i think i will. The problem is that the database isn't currently in the nearline server but on some testing node. It will soon be in the nearline db so this can wait or it can be changed when the (simple) migration is complete. What do you think ?

Reply:
I would say go ahead and change it. For testing purposes you can edit the variables in your settings file to point to the test node. For example:

```
DB_HOST_NEARLINE='test node ip address'
DB_NAME_NEARLINE='database name'
DB_PORT_NEARLINE=whatever port you're using
```

Reply:
Simple question when you say settings file where would i put that. I don't have one yet.

Reply:
The README should have information on how to set up a local copy of minard. Basically it's just a configuration file that contains lots of things like ip addresses and directory locations.

Reply:
Oh sorry i didn't mean that, let me say it again.  I had wanted to use my current db which is on some digital ocean node to fill the information on the page. I only wanted to do this until the table has been ported to the nearline server. Which can be done soon. So if you feel strongly about it not using an external server to provide data for the page then this can wait until i change things. Otherwise that's why i'm setting up my own connection.

Reply:
Data is now in the nearline database, so i have made the change to use the native psql engine.